docs/extras/integrations/tools/metaphor_search.ipynb

```diff
@@ -72,14 +72,14 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
+   "execution_count": 5,
    "metadata": {},
    "outputs": [],
    "source": [
     "@tool\n",
     "def search(query: str):\n",
-    "    \"\"\"Call search engine with a query.\"\"\"\n",
-    "    return client.search(query)\n",
+    "    \"\"\"Call search engine with a query. Use autoprompt by default to convert query to a Metaphor query.\"\"\"\n",
```

Comment:
i actually think its BAD to put this in the docstring - the agent sees this docstring and this isnt a super relevant fact and could distract from the true parameters

Could add that note in a comment

speaking of, may be good to add a comment about docstrings mattering haha

Reply:
Ah yes good point - I haven't played with this tool decorator syntax before, but obviously makes sense why that is the case. I guess I would like to have one comment that is the docstring and another that describes what the docstring does haha.